arm-monitor/2017-04-01/swagger/activityLogAlerts_API.json

```diff
@@ -0,0 +1,451 @@
+                "x-ms-examples": {
+                    "List activity log alerts": {
+                        "$ref": "../examples/listActivityLogAlerts.json"
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/ActivityLogAlertList"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the list of activity log alerts could not be retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/activityLogAlerts": {
+            "get": {
+                "description": "Get a list of all activity log alerts in a resource group.",
+                "operationId": "ActivityLogAlerts_ListByResourceGroup",
+                "x-ms-examples": {
+                    "List activity log alerts": {
+                        "$ref": "../examples/listActivityLogAlerts.json"
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/ActivityLogAlertList"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the list of activity log alerts could not be retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Resource": {
```

Comment:
Thats right, an [example](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-recoveryservices/2016-06-01/swagger/vaultusages.json#L46) would serve you better though